pango/Gir.toml

```diff
@@ -33,8 +33,6 @@ generate = [
     "Pango.CoverageLevel",
     "Pango.Direction",
     "Pango.EllipsizeMode",
-    "Pango.EngineLang",
```

Comment:
Shouldn't those be moved to the ignore list? otherwise gir -m not_bound will complain about them i guess?